react/src/lib/components/DeckGLMap/layers/map/mapLayer.stories.tsx

```diff
@@ -28,7 +28,7 @@ const meshMapLayer = {
         rotDeg: 0,
     },
     propertiesUrl: "kh_netmap_5_m.float32",
-    contours: [0, 100.0],
+    contours: [0, 1000],
```

Comment:
I think the previous contour interval is more useful.

Reply:
fixed